packages/backend/src/api/user/userRouter.ts

```diff
@@ -1,44 +1,113 @@
 import { OpenAPIRegistry } from '@asteasolutions/zod-to-openapi';
-import express, { Request, Response, Router } from 'express';
+import bodyParser from 'body-parser';
+import express, { NextFunction, Request, Response, Router } from 'express';
+import authenticate from 'src/common/middleware/authConfig/authentication';
+import authorizeRole from 'src/common/middleware/authConfig/authorizeRole';
+import passport from 'src/common/middleware/authConfig/passport';
+import { Roles } from 'src/common/middleware/authConfig/roles';
+import { verifyUser } from 'src/common/middleware/verifyUser';
 import { z } from 'zod';
 
 import { createApiResponse } from '../../api-docs/openAPIResponseBuilders';
 import { handleServiceResponse, validateRequest } from '../../common/utils/httpHandlers';
 import { GetUserSchema, UserSchema } from '../user/userModel';
 import { userService } from '../user/userService';
+import {
+  CreateUserRequest,
+  CreateUserSchema,
+  DeleteUserRequest,
+  DeleteUserSchema,
+  GetUserRequest,
+  GetUsersRequest,
+  GetUsersSchema,
+  LoginUserRequest,
+  LoginUserSchema,
+  UpdateUserRequest,
+  UpdateUserSchema,
+  UserRefreshRequest,
+  UserRefreshTokenSchema,
+} from './userRequest';
 
 export const userRegistry = new OpenAPIRegistry();
 
 userRegistry.register('User', UserSchema);
 
 export const userRouter: Router = (() => {
   const router = express.Router();
-
+  router.use(bodyParser.json());
```

Comment:
why do we need this? we already have app.use(express.json()); in the server.ts file

Reply:
https://stackoverflow.com/questions/47232187/express-json-vs-bodyparser-json